gallery/src/components/NxDrawer/NxDrawerWithNxFormExample.tsx

```diff
@@ -0,0 +1,224 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React, { useState, ChangeEvent, FormEvent } from 'react';
+
+import {
+  NxDrawer,
+  NxButton,
+  NxP,
+  NxBackButton,
+  NxPageMain,
+  NxPageTitle,
+  NxH1,
+  NxFontAwesomeIcon,
+  NxForm,
+  NxFormGroup,
+  NxFieldset,
+  NxCheckbox,
+  NxTextInput,
+  nxTextInputStateHelpers,
+  useToggle,
+  NxRadio,
+  NxStatefulTextInput,
+  nxFormSelectStateHelpers,
+  NxFormSelect,
+  NxToggle
+} from '@sonatype/react-shared-components';
+
+import { faEdit } from '@fortawesome/free-solid-svg-icons';
+
+export default function NxDrawerWithNxFormExample() {
+  const [showDrawer, setShowDrawer] = useState(false),
+      [showDrawerOverflowing, setShowDrawerOverflowing] = useState(false),
+      [usernameState, setUsernameState] = useState(nxTextInputStateHelpers.initialState('')),
+      [passwordState, setPasswordState] = useState(nxTextInputStateHelpers.initialState('')),
+      [isRed, toggleRed] = useToggle(false),
+      [isBlue, toggleBlue] = useToggle(false),
+      [isGreen, toggleGreen] = useToggle(false);
+
+  const [color, setColor] = useState<string | null>(null);
```

Comment:
We want to try to avoid having complex parts to our examples that are besides the main point of that particular example.  For instance, this example is about drawers, not form logic, so I'd make the form part as simple as possible so that the reader can focus on the drawer part.  The form can just be a series of stateful text inputs, checkboxes, etc.

Reply:
> We want to try to avoid having complex parts to our examples that are besides the main point of that particular example. For instance, this example is about drawers, not form logic, so I'd make the form part as simple as possible so that the reader can focus on the drawer part. The form can just be a series of stateful text inputs, checkboxes, etc.

Fixed here: 4457934e2464b8b0ea4838871e2bb78c972214ab